ext/RMagick/rmpixel.c

```diff
@@ -631,7 +672,11 @@ Pixel_from_MagickPixel(const MagickPixel *pp)
     pixel->red     = ROUND_TO_QUANTUM(pp->red);
     pixel->green   = ROUND_TO_QUANTUM(pp->green);
     pixel->blue    = ROUND_TO_QUANTUM(pp->blue);
+#if defined(IMAGEMAGICK_7)
+    pixel->alpha   = ROUND_TO_QUANTUM(pp->alpha);
```

Comment:
Unrelated to this PR, but would it be possible to write `ROUND_TO_QUANTUM` as a util function instead of a `define`? `rm_round_to_quantum(pp->alpha)`.

Reply:
We could do that and it woul probably make it more readable. Something for another PR?

Reply:
Yeah, these can be done separately. It was more for my understanding.